docs/catalog-restrictions.md

```diff
@@ -190,7 +186,7 @@ spec:
         namespace: brokers
   catalogRestrictions:
     serviceClass:
-    - "name!=AABBB-CCDD-EEGG-HIJK"
+    - "name!=MySQL"
```

Comment:
should this be `spec.externalName` like you have in the servicePlan section below? Doesn't `name` refer to the k8s.name ?